map_gui/src/colors.rs

```diff
@@ -172,8 +174,19 @@ impl ColorScheme {
         cs
     }
 
+    fn pregame() -> ColorScheme {
```

Comment:
Secretly this is basically the "legacy day theme". It lives on in pregame only for now.